book/src/pong_tutorial/pong_tutorial_01.md

```diff
@@ -0,0 +1,139 @@
+# Opening (and closing!) a window
+
+Let's start a new project:
+
+`amethyst new pong`
+
+If you run this project with `cargo run`, you'll end up with a window titled "pong" that renders a really delightful shade of green. Press `Esc` to quit.
+
+We've opened and closed a window, so we're basically done! But let's write this functionality ourselves so we're sure we know what's going on.
+
+In `src` there's a `main.rs` file. Delete everything, then add these imports:
+
+```rust,ignore
+extern crate amethyst;
+
+use amethyst::prelude::*;
+use amethyst::renderer::{DisplayConfig, DrawFlat, Event, KeyboardInput,
+                         Pipeline, PosNormTex, RenderBundle, RenderSystem,
+                         Stage, VirtualKeyCode, WindowEvent};
+```
+
+We'll be learning more about these as we go through this tutorial. The prelude includes the basic (and most important) types like Application, World, and State.
+
+Now we create our core game struct:
+
+```rust,ignore
+struct Pong; 
+```
+
+We'll be implementing the [State][st] trait on this struct, which is used by Amethyst's state machine to start, stop, and update the game. But for now we'll just implement one method: 
```

Comment:
I think that @ebkalderon (the owner of Amethyst) once wrote a guideline which said that there should be a manual line break, but I'm not 100% sure.

Reply:
Ah yeah, contributing.md says 80 characters, didn't know that applied to prose.